### PR TITLE
Adding inputs to time picker

### PR DESCRIPTION
**Pull request by palbizu**
## Description
Adding new inputs in TimePicker:
- Disabled : boolean
- Display Mode : DisplayMode (with-border/with-background)
- IconSize: defines chevron size (Small/Extra-small)

### Testing
Visual Testing

image

image

image

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: projects/components/src/multi-select/multi-select.component.ts [hunk 1]
```diff
@@ -63,7 +63,11 @@ import { MultiSelectJustify } from './multi-select-justify';
                 <span *ngIf="triggerValues.selectedItemsCount > 1" class="trigger-more-items"
                   >+{{ triggerValues.selectedItemsCount - 1 }}</span
                 >
-                <ht-icon class="trigger-icon" icon="${IconType.ChevronDown}" size="${IconSize.Small}"></ht-icon>
+                <ht-icon
+                  class="trigger-icon"
+                  icon="${IconType.ChevronDown}"
+                  [size]="this.size === '${IconSize.ExtraSmall}' ? '${IconSize.ExtraSmall}' : '${IconSize.Small}'"
```
Comment (arjunlalb) [inline projects/components/src/multi-select/multi-select.component.ts:69]: why not use `this.size` directly?

Comment (palbizu) [reply projects/components/src/multi-select/multi-select.component.ts:69]: because I would be modifying the regular case that already uses Small size as a regular

Comment (anandtiwary) [reply projects/components/src/multi-select/multi-select.component.ts:69]: `[size]="this.size"` should also work right?

Comment (anandtiwary) [reply projects/components/src/multi-select/multi-select.component.ts:69]: hmm.. you are comparing and assigning`SelectSize` with IconSize. that seems incorrect.

Comment (anandtiwary) [reply projects/components/src/multi-select/multi-select.component.ts:69]: This component also has iconSize btw.

File: projects/components/src/time-picker/time-picker.component.ts [hunk 2]
```diff
@@ -9,8 +9,8 @@ import { PredefinedTimeService } from '../time-range/predefined-time.service';
   styleUrls: ['./time-picker.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="time-picker">
-      <ht-popover class="time-selector" [closeOnClick]="true">
+    <div class="time-picker" [ngClass]="[this.disabled ? 'disabled' : '']">
```
Comment (anandtiwary) [inline projects/components/src/time-picker/time-picker.component.ts:12]: `[ngClass]="{'disabled': this.disabled}"`

File: projects/components/src/form-field/form-field.component.ts [hunk 3]
```diff
@@ -65,6 +65,9 @@ export class FormFieldComponent {
   @Input()
   public showBorder: boolean = false;
 
+  @Input()
+  public showOnlyErrorBorder: boolean = false;
```
Comment (anandtiwary) [inline projects/components/src/form-field/form-field.component.ts:69]: why this change?

Comment (palbizu) [reply projects/components/src/form-field/form-field.component.ts:69]: To show only red borders

Comment (palbizu) [reply projects/components/src/form-field/form-field.component.ts:69]: To show only red borders

Comment (anandtiwary) [reply projects/components/src/form-field/form-field.component.ts:69]: We need consistent behavior across forms. Why do we want to show borders but not error message?

Comment (palbizu) [reply projects/components/src/form-field/form-field.component.ts:69]: Yes, I agree but exist some cases where it would be good to show the red borders without messages, for example when I want to show two nested fields with the same message.

File: projects/components/src/multi-select/multi-select.component.ts (hunk 1 above)
Comment (palbizu) [reply projects/components/src/multi-select/multi-select.component.ts]: thanks, done

File: projects/components/src/time-picker/time-picker.component.ts (hunk 2 above)
Comment (palbizu) [reply projects/components/src/time-picker/time-picker.component.ts]: done

File: projects/components/src/form-field/form-field.component.ts (hunk 3 above)
Comment (anandtiwary) [reply projects/components/src/form-field/form-field.component.ts:69]: In that case, we should be combining two fields inside the same form-field component

Comment (palbizu) [reply projects/components/src/form-field/form-field.component.ts:69]: So should I use an object (with a pair of variables inside) instead of showing the input separatly?

Comment (palbizu) [reply projects/components/src/form-field/form-field.component.ts:69]: showOnlyErrorBorder  and showTransparentBackground were Removed

**Review by jake-bassett — APPROVED** [review]
